src/main/resources/synthea.properties

```diff
@@ -14,11 +14,11 @@ exporter.fhir.export = true
 exporter.fhir_stu3.export = false
 exporter.fhir_dstu2.export = false
 exporter.fhir.use_shr_extensions = false
-exporter.fhir.use_us_core_ig = true
+exporter.fhir.use_us_core_ig = false
 exporter.fhir.transaction_bundle = true
 exporter.fhir.bulk_data = false
 exporter.groups.fhir.export = false
-exporter.hospital.fhir.export = true
+exporter.hospital.fhir.export = false
```

Comment:
I think you want to revert these changes.

US Core should be default, so should the hospital export.